src/mgmt/rpc.h

```diff
@@ -204,4 +207,23 @@ CRT_GEN_STRUCT(server_entry, DAOS_SEQ_SERVER_ENTRY);
 CRT_RPC_DECLARE(mgmt_tgt_map_update, DAOS_ISEQ_MGMT_TGT_MAP_UPDATE,
 		DAOS_OSEQ_MGMT_TGT_MAP_UPDATE)
 
+/* List pools: returns an array of mgmt_list_pools_one */
+#define DAOS_SEQ_MGMT_LIST_POOLS_ONE \
+	((uuid_t)		(lp_puuid)	CRT_VAR) \
+	((d_rank_list_t)	(lp_svc)	CRT_PTR)
+
+CRT_GEN_STRUCT(mgmt_list_pools_one, DAOS_SEQ_MGMT_LIST_POOLS_ONE);
+
+#define DAOS_ISEQ_MGMT_LIST_POOLS /* input fields */		 \
+	((d_string_t)		(lp_grp)		CRT_VAR) \
+	((uint64_t)		(lp_npools)		CRT_VAR)
+
+#define DAOS_OSEQ_MGMT_LIST_POOLS /* output fields */			   \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
I would like to disregard this style warning in this patch because the macro definition is in the same style as the others in this file.